scripts/test_all.sh

```
@@ -1,13 +1,13 @@
-#!/bin/sh
-set -eu
+#!/bin/bash
+set -euo pipefail
 
 go install github.com/jstemmer/go-junit-report@latest
 
 for test in $(dirname $0)/test_*; do
```

Shellcheck (0.11.0)

[info] Quote expansions in this for loop glob to prevent wordsplitting, e.g. "$dir"/*.txt .

(SC2231)